README.md

```diff
@@ -1,7 +1,78 @@
-Instagram Challenge
+
+Anthony's Instagram Challenge
 ===================
 
-## Instructions
+## Introduction
```

Comment:
A well worked challenge. I was able to setup easily following the README. Instructions were very clear and easy to follow. I was able to successfully sign in and log in. Error messages worked when using incorrect information. I could create a post. Great work!